src/hooks/useForkTools.ts

```diff
@@ -40,7 +38,12 @@ const useForkTools = (id?: string): ForkTools => {
     const currentBlockNumber = await provider.getBlockNumber();
     const resp = await axios.post(
       forkAPI,
-      { network_id: 1, block_number: currentBlockNumber },
+      { network_id: 1, 
+        block_number: currentBlockNumber,
+        chain_config: { 
+          chain_id: 1277971
```

Comment:
can we or should we make the fork chain id an env variable?

Reply:
Could do so, I think better would be to simply append 127797.. to the current netwrok_id in this case.